### PR TITLE
Added "What's-This" help to launcher creator

### PR DESCRIPTION
**Pull request by tsujan**

### CODE REVIEW

File: pcmanfm/desktopentrydialog.ui [hunk 1]
```diff
@@ -58,6 +61,11 @@
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="descriptionEdit">
+       <property name="whatsThis">
+        <string>A short description or generic name, like &quot;Web Browser&quot; or &quot;Home Directory&quot;.
+
```
Comment (stefonarch) [inline pcmanfm/desktopentrydialog.ui:66]: "It will be shown as tooltip" ?

File: pcmanfm/desktopentrydialog.ui [hunk 2]
```diff
@@ -187,6 +209,13 @@
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="catEdit">
+       <property name="whatsThis">
+        <string>Semicolon separated categories in which the entry should be shown if it is inside the DE's main menu.
```
Comment (stefonarch) [inline pcmanfm/desktopentrydialog.ui:213]: inside the DE's main menu (usually saved in ~/.local/share/applications).  ?

File: pcmanfm/desktopentrydialog.ui (hunk 1 above)
Comment (tsujan) [reply pcmanfm/desktopentrydialog.ui:66]: Not necessarily. It depends on the DE and its main menu.

File: pcmanfm/desktopentrydialog.ui (hunk 2 above)
Comment (tsujan) [reply pcmanfm/desktopentrydialog.ui:213]: Yes. Why did you ask?

Comment (stefonarch) [reply pcmanfm/desktopentrydialog.ui:213]: I asked if we should also tell _how_ to add it to the menu, adding the path to the location in the text.

Comment (tsujan) [reply pcmanfm/desktopentrydialog.ui:213]: Oh, that's really beyond this small utility. And if someone adds "Categories", they probably know what they'll do with it.

**Comment by tsujan** [comment]
Added a small hint on `~/.local/share/applications`. We don't want a full tutorial about menu editing here ;)

**Comment by stefonarch** [comment]
Oh, we could put  also some  links to tutorials on youtube, they have nice music usually!  ;)
Anyway, I declare myself satisfied :)

GTM

**Pull request merged**